README.md

```diff
@@ -40,14 +40,13 @@ Kepler instances are configured by the [kepler.toml](kepler.toml) configuration
 
 The following common options are available:
 
-| Option               | env var                     | description                                                    |
-|:---------------------|:----------------------------|:---------------------------------------------------------------|
-| log_level            | KEPLER_LOG_LEVEL            | Set the level of logging output, options are "normal", "debug" |
-| address              | KEPLER_ADDRESS              | Set the listening address of the kepler instance               |
-| port                 | KEPLER_PORT                 | Set the listening TCP port for the kepler instance             |
-| storage.blocks.type  | KEPLER_STORAGE_BLOCKS_TYPE  | Set the mode of block storage, options are "Local" and "S3"    |
-| storage.indexes.type | KEPLER_STORAGE_INDEXES_TYPE | Set the type of the index store, options are "Local" and "DynamoDB" |
-| orbits.allowlist     | KEPLER_ORBITS_ALLOWLIST     | Set the URL of an allowlist service for gating the creation of Orbit Peers                                                               |
+| Option               | env var                     | description                                                                |
+|:---------------------|:----------------------------|:---------------------------------------------------------------------------|
+| log_level            | KEPLER_LOG_LEVEL            | Set the level of logging output, options are "normal", "debug"             |
+| address              | KEPLER_ADDRESS              | Set the listening multi-address of the kepler instance                     |
```

Comment:
I'm fairly certain these two env vars don't do anything, they should be `RUST_LOG` or `ROCKET_LOG`, and `ROCKET_ADDRESS`. It also means you can't pass a multi-address for the HTTP server. But I might be wrong

Reply:
Also, remember to update the Dockerfile to bind the relay to `0.0.0.0` otherwise it won't be available

Reply:
my bad, confused the rocket params with the kepler ones, I put the rocket address/port back and added readme/kepler.toml example entries for `relay.addresses` (now a vector of multiaddrs).

Reply:
Seems like only `RUST_LOG` works, `ROCKET_LOG` expects our `Logging` config struct. in `main` we accept any config stuff with either `ROCKET_*` or `KEPLER_*`, so `ROCKET_LOG` as an expected simple string gets over-ridden by our logging struct